packages/mdc-drawer/mdc-drawer.scss

```diff
@@ -20,6 +20,108 @@
 // THE SOFTWARE.
 //
 
-@import "./common";
-@import "./dismissible/mdc-drawer-dismissible";
-@import "./modal/mdc-drawer-modal";
+@import "@material/side-sheet/mixins";
+@import "@material/side-sheet/variables";
+@import "@material/typography/mixins";
+@import "./mixins";
+@import "./variables";
+
+// postcss-bem-linter: define drawer
+
+.mdc-drawer {
+  @include mdc-side-sheet-base_;
+  @include mdc-side-sheet-transitions_(leading);
+  @include mdc-side-sheet-outline-width_(1px, leading);
+  @include mdc-drawer-title-ink-color($mdc-drawer-title-ink-color);
+  @include mdc-drawer-subtitle-ink-color($mdc-drawer-subtitle-ink-color);
+  @include mdc-drawer-outline-color($mdc-drawer-divider-color);
+  @include mdc-drawer-surface-fill-color($mdc-drawer-surface-fill-color);
+  @include mdc-drawer-item-icon-ink-color($mdc-drawer-item-inactive-ink-color);
+  @include mdc-drawer-item-text-ink-color($mdc-drawer-item-inactive-ink-color);
+  @include mdc-drawer-item-activated-icon-ink-color($mdc-drawer-item-activated-ink-color);
+  @include mdc-drawer-item-activated-text-ink-color($mdc-drawer-item-activated-ink-color);
+  @include mdc-drawer-shape-radius(large);
+  @include mdc-drawer-item-shape-radius(4px);
+  @include mdc-drawer-z-index($mdc-drawer-z-index);
+  @include mdc-drawer-width($mdc-drawer-width);
+
+  // postcss-bem-linter: ignore
+  .mdc-list-item {
+    @include mdc-typography(subtitle2);
+
+    height: calc(48px - 2 * #{$mdc-drawer-list-item-spacing});
+    // To accomodate margin conflict.
+    margin: ($mdc-drawer-list-item-spacing * 2) 8px;
+    padding: 0 $mdc-drawer-surface-padding / 2;
+  }
+
+  // postcss-bem-linter: ignore
+  .mdc-list-item:nth-child(1) {
+    margin-top: 2px;
+  }
+
+  // postcss-bem-linter: ignore
+  .mdc-list-item:nth-last-child(1) {
+    margin-bottom: 0;
+  }
+
+  // postcss-bem-linter: ignore
+  .mdc-list-group__subheader {
+    @include mdc-typography(body2);
+    @include mdc-typography-baseline-top(24px);
+
+    margin: 0;
+    padding: 0 $mdc-drawer-surface-padding;
+  }
+
+  // postcss-bem-linter: ignore
+  .mdc-list-divider {
+    margin: 3px 0 4px 0;
+  }
+
+  // Prevents list item children from being included in the click target.
+  // postcss-bem-linter: ignore
+  .mdc-list-item__text,
+  .mdc-list-item__graphic {
+    pointer-events: none;
+  }
+}
+
+.mdc-drawer__header {
+  flex-shrink: 0;
+  box-sizing: border-box;
+  min-height: 64px; // same as $mdc-top-app-bar-row-height
```

Comment:
Would it make sense to use the value of `$mdc-top-app-bar-row-height` directly?

Reply:
I suspect we didn't do that to avoid adding a hard dependency on the top-app-bar package, when drawer can be used with or without one.

Reply:
If we're only importing `_variables.scss`, why does it matter?

It won't output any additional CSS, and it has the advantage that we don't have to remember to keep two magic values in sync.

What's more, `@material/top-app-bar` only adds `736K` to `node_modules` ([I checked](https://gist.github.com/acdvorak/706227c1458cfbff229e313c2a8ebc31)).
NPM dependency trees are typically _massive_; `736K` is barely even noticeable.

Personally, I would prioritize maintainability over NPM dependency size.

Reply:
Yeah, that seems fair, I'll add it.